web/packages/core/texts/cs-CZ/messages.ftl

```diff
@@ -14,6 +14,15 @@ enable-hardware-acceleration = Zdá se, že hardwarová akcelerace není povolen
 view-error-details = Zobrazit podrobnosti o chybě
 open-in-new-tab = Otevřít na nové kartě
 click-to-unmute = Kliknutím zrušíte ztlumení
+clipboard-message-title = Kopírování a vkládání v Ruffle
+clipboard-message-description =
+    { $variant ->
+       *[unsupported] Váš prohlížeč nepodporuje plný přístup ke schránce,
+        [access-denied] Přístup ke schránce byl odepřen,
+    }, ale místo toho můžete vždy použít tyto zkratky:
```

Comment:
I believe a comma will be doubled here, as it's also present above in both variants.

Reply:
good point, nice catch!

Reply:
It's not a good point, it's a... _bad comma_ :eyes:

...

I'll see myself out